docs/docs/intro.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 1
+---
+
+# Welcome to Scaffold Stellar!
+
+**Scaffold Stellar** is a developer toolkit for building decentralized applications (dApps) and smart contracts on the [**Stellar** blockchain](https://stellar.org).
+
+It helps you go from **idea** to **working full-stack dApp** faster — by providing CLI tools, reusable contract templates, a smart contract registry, and a modern frontend.
+
+---
+
+## Why Use Scaffold Stellar?
+
+- Simplifies blockchain dApp development
+- Generates smart contract projects and React UIs
+- Deploys smart contracts and manages versions
+- Easy to learn for newcomers; powerful for pros
+
+---
+
+## What Is Stellar?
+
+[**Stellar**](https://www.stellar.org/) is a blockchain designed for fast, low-cost financial transactions and smart contracts written in **Rust** and compiled to **WebAssembly (Wasm)**.
+
+With Scaffold Stellar, you write smart contracts in Rust and interact with them using modern TypeScript + React tooling.
+
+---
+
+## Additional Developer Resources
+
+- Video: [Intro to Scaffold Stellar](https://www.youtube.com/watch?v=559ht4K4pkM)
+- Video: [Which Frontend?](https://www.youtube.com/watch?v=pz7O54Oia_w)
+- Video: [Get Started Building](https://www.youtube.com/watch?v=H-M962aPuTk)
+- Video: [Live Demo of Scaffold Stellar](https://www.youtube.com/watch?v=0syGaIn3ULk) 👈 Start Here
```

Comment:
This video is 3 months old and I assume quite out-of-date by now; is it really the best place to start?

Reply:
Now we have your video from Meridian. Maybe that's a better start? Or we wait for the tutorial videos.

Reply:
Fixable later

Reply:
My comment to @chadoh is that I don't want feedback on this PR to be about editing individual existing docs -- fix those in other PRs. Let's get this website up!